nilearn/maskers/base_masker.py

```diff
@@ -9,13 +9,18 @@
 from sklearn.base import BaseEstimator, TransformerMixin
 from sklearn.utils.estimator_checks import check_is_fitted
 
+from nilearn import _utils, image, masking, signal
+from nilearn._utils import logger, stringify_path
```

Comment:
Instead of importing whole modules, that would be better to import individual functions from modules.